docs/plotting_roms_output.ipynb

```diff
@@ -112,10 +112,18 @@
   },
```

Comment:
<p>Final nit: Should we add a period after the sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CWorthy-ocean/roms-tools/pull/366/discussion/#comment-2178539449'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>